src/components/CvssCalculator/CvssFactorBtns.vue

```diff
@@ -0,0 +1,170 @@
+<script setup lang="ts">
```

Comment:
Could we change the name to `CvssFactorButtons`.vue?

Reply:
Sure